main.go

```diff
@@ -33,30 +38,54 @@ func run(args []string, stdout io.Writer) error {
 	if len(args) == 0 {
 		return errors.New("no arguments")
 	}
+
 	flags := flag.NewFlagSet(args[0], flag.ExitOnError)
-	flag.BoolVar(&debug, "debug", false, "sets log level to debug and console pretty output")
+
+	debug := flag.Bool("debug", false, "sets log level to debug and console pretty output")
+
+	// (&debug,
+	// 	"debug",
+	// 	false,
+	// 	"sets log level to debug and console pretty output")
 
 	if err := ff.Parse(flags, args); // ff.WithEnvVarNoPrefix(),
 	// ff.WithConfigFileFlag("config"),
 	// ff.WithConfigFileParser(fftoml.Parser),
 	err != nil {
 		return err
 	}
-	// Logger.Configure(Logger.Config{
-	// 	ConsoleLoggingEnabled: true,
-	// })
-	// l.Log.Info().Msg("func run() completed")
-	// Logger.Info().Msg("func run() completed")
+
+	LogLevel := "info"
+	if *debug {
+		LogLevel = "debug"
+	}
+
+	c := logger.Config{
+		Enable:                true,
+		ConsoleLoggingEnabled: true,
+		EncodeLogsAsJson:      false,
+		FileLoggingEnabled:    false,
+		Directory:             "",
+		Filename:              "",
+		MaxSize:               MaxSize,
+		MaxBackups:            MaxBackups,
+		MaxAge:                MaxAge,
+		Level:                 LogLevel,
+	}
+
+	_ = logger.InitLogger(c)
+
 	return nil
 }
 
-// func main() {
-// 	root := &ffcli.Command{
-// 		Exec: func(ctx context.Context, args []string) error {
-// 			println("hello world")
-// 			return nil
-// 		},
-// 	}
+// CountViolations counts the number of lines that would need to be fixed by adding semantic line break. It returns an integer value of the violation count found.
+func CountViolations(content []byte) int {
+	re := regexp.MustCompile(`(?is)^.*\w\.\s\w.*$`)
+	matches := re.FindAllString(string(content), -1)
+	logger.Log.Info().Int("ViolationCount", len(matches)).Msg("CountViolations")
```

Comment:
This works now as logger is initialized and passing through the pointer to zerolog.